realm/realm-library/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -564,7 +564,8 @@ public void looperThreadQuitsLooperEarly() throws InterruptedException {
 
         // Starts background looper and let it hang.
         ExecutorService executorService = Executors.newSingleThreadExecutor();
-        executorService.submit(new Runnable() {
+        //noinspection unused
+        final Future<?> future = executorService.submit(new Runnable() {
```

Comment:
Of course we can suppress the ErrorProne warning directly, but it affects entire method.
I think suppressing scope should be as narrow as possible.